sale_automatic_workflow/models/sale_workflow_process.py

```diff
@@ -35,36 +35,41 @@ def _default_filter(self, xmlid):
         default='direct',
     )
     validate_order = fields.Boolean(string='Validate Order')
-    order_filter_domain = fields.Char(
+    order_filter_domain = fields.Text(
         string='Order Filter Domain',
-        default="[('state', '=', 'draft')]"
+        related='order_filter_id.domain',
```

Comment:
@lmignon - Oh interesting, that makes complete sense thank you for elaborating. 

I think in the context here we're good, but I am going to do some more investigation on the implications of this.